coreos-ostree-importer/coreos_ostree_importer.py

```diff
@@ -160,6 +160,11 @@ def process(self, message: fedora_messaging.api.Message):
                 branch=ostree_ref,
             )
 
+        # Update the summary file if we're in the prod repo. The compose
+        # repo doesn't use a summary file and creating one causes problems
```

Comment:
Optional: link to more background on this?

Reply:
unfortunately I don't really have one :( I guess I could try to point to some of the code that needs this. ENOTIME for now